pepper-apis/docs/specification/src/endpoints/user.studies.activities.yml

```diff
@@ -29,13 +29,50 @@ post:
   tags:
     - Activities
   summary: create a new instance of an activity
-  description: given an `activityCode`, the server will create a new instance of the activity for the specified user and return the `instanceGuid` of the newly created default instance.
+  description: Given an `activityCode`, the server will create a new instance of the activity for the specified user if the configured conditions for the activity is met.
   requestBody:
-    $ref: '../pepper.yml#/components/requestBodies/ActivityCreateRequestBody'
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          required:
+            - activityCode
+          properties:
+            activityCode:
+              description: activity to create a new instance from
+              type: string
+              example: ABCDEFGH12
   responses:
       200:
-        $ref: '../pepper.yml#/components/responses/ActivityCreateResponse'
+        description: successfully created a new instance of the specified activity
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - instanceGuid
+              properties:
+                instanceGuid:
+                  type: string
+                  example: ABCDEFGH12
+      400:
```

Comment:
I have cleaned up the error responses, so I will consider this a breaking change.